templates/confirmation-email-sent.html

```diff
@@ -0,0 +1,30 @@
+{% extends 'layouts/_base.html' %}
+{% from 'components/panel/_macro.njk' import onsPanel %}
+{% set hide_signout_button = content.hide_signout_button %}
+{% set sign_out_url = content.sign_out_url %}
+{% set page_title = _("Confirmation email sent") ~ " - " ~ _("Census 2021") %}
+
+{% block main %}
+   {% call onsPanel({
+     "spacious": true,
+     "type": "success",
+     "classes": "u-mb-s"
+     })
+   %}
+  <h1 data-qa="confirmation-text">{{ _("A confirmation email has been sent to {email}").format(email = content.email)  }}</h1>
+ {% endcall %}
+
+  <p><strong>{{ _("Didn't receive an email?")  }}</strong></p>
+  <p>{{ _("It can take a few minutes for the email to arrive. If it doesn't arrive, check your junk mail, or you can
+    <a href='{url}' id='send-another-email'>send another confirmation email</a>.</p>").format(url = content.url)  }}
+
+
+{% call onsPanel({
+    "id": "submission-warning",
+    "type": "warn"
+    })
+  %}
+    {{ _("Make sure you <a href='{url}'>leave this page</a> or close your browser if using a shared device").format(url = sign_out_url)  }}
```

Comment:
The `sign_out_url` seems unnecessary.
```suggestion
    {{ _("Make sure you <a href='{url}'>leave this page</a> or close your browser if using a shared device").format(url = content.sign_out_url)  }}
```